Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM library/python:3.7-alpine
+
+LABEL maintainer OSG Software <help@opensciencegrid.org>
+
+COPY *.py /usr/local/bin/
+
+ENTRYPOINT ["/usr/local/bin/group_fixup.py", "--fix-all"]
```

Comment:
Let's avoid a default entrypoint and just specify the command + options in the k8s manifest https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#define-a-command-and-arguments-when-you-create-a-pod